### DISCUSSION TITLE
Training vs. validation speed

### DISCUSSION THREAD

**Discussion by pytholic** [TorchIO-project/torchio#888] [General]
Hi and thank you for the repo!

My training set is providing approximately `8it/s` while the validation set is giving `30it/s` which is almost 3x.
Is it normal behavior? If so then what is the reason for that.

If not then what can be the issue here?

**Comment by fepegar** [comment]
Hi, @pytholic. It's probably normal. It depends on many things such as the size of your training and validation sets, but an important difference is that you don't perform backpropagation during validation.
